### DISCUSSION TITLE
Mjpeg over rtp

### DISCUSSION THREAD

**Discussion by nahueltaibo** [ngraziano/SharpRTSP#109] [Q&A]
I'm trying to create a simple fpv like camera feed for a robot in c#.

I already have the jpeg frames I geet from a webcam using emgu cv library.
Is there any example of how to feed this jpeg frames to the rtp session? (I know about the client example receiving jpeg frames, but I'm missing how to send them from the server)

Sorry If the question is too basic, I'd  appreciate if you can guide me on reading material to become better at real time streaming in general :😀

**Comment by Revan1985** [comment]
I have no ready any example for server jpeg, but you can read the following rfc for the official documentation
[RFC-2435](https://www.rfc-editor.org/rfc/rfc2435)
You should be able to understood how it works...

**Comment by ngraziano** [comment]
Hi,
I have try to add and example to serve MJPEG but there is some problem with the RTP encoding
https://github.com/ngraziano/SharpRTSP/tree/dotnetcore/RtspCameraMJPEGExample
It play with ffplay but not with VLC, I must have done an error but it is a base for an exemple to send mjpeg.
